source/opt/def_use_manager.cpp

```diff
@@ -103,24 +105,59 @@ bool DefUseManager::ReplaceAllUsesWith(uint32_t before, uint32_t after) {
 
 void DefUseManager::ClearDef(uint32_t def_id) {
   if (id_to_def_.count(def_id) == 0) return;
```

Comment:
Fixed in 4d7c019.

And it doesn't call ClearInst() anymore, so for instructions with result_id, we won't look for the maps twice.